crates/wasmtime/src/runtime.rs

```diff
@@ -193,10 +194,15 @@ impl Config {
         self.validating_config
             .operator_config
             .enable_reference_types = enable;
-        // The reference types proposal depends on the bulk memory proposal
+        self.flags
+            .set("enable_safepoints", if enable { "true" } else { "false" })
+            .unwrap();
```

Comment:
```suggestion
        if enable {
            self.flags.enable("enable_safepoints").unwrap();
        }
```

Reply:
This doesn't work for disabling reference types, and there is no `disable` function to go along with `enable` either.